src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/components/ApplicantListItem/index.js

```diff
@@ -17,178 +17,165 @@ import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
-        personClassJobTitle = profile.Class;
-      }
-      // set job titles up
-    } else {
-      personType = 'Not Student';
-      if (profile.JobTitle !== undefined) {
-        personClassJobTitle = profile.JobTitle;
-      }
-    }
-
-    return (
-      <ListItem
-        key={profile.AD_Username}
-        component={Link}
-        target="_blank"
-        to={`/profile/${profile.AD_Username}`}
-        className={'list-item'}
-      >
-        <ListItemAvatar>
-          {this.state.avatar ? (
-            <Avatar
-              className={`avatar`}
-              src={`data:image/jpg;base64,${this.state.avatar}`}
-              alt=""
-            />
-          ) : (
-            <Avatar>
-              <PersonIcon color="primary" />
-            </Avatar>
-          )}
-        </ListItemAvatar>
-        <Grid container alignItems="center" spacing={3}>
-          <Grid item xs={6}>
-            <ListItemText
-              primary={nickname ? fullName.replace(' ', ' ' + nickname + ' ') : fullName}
-              secondary={profile.AD_Username}
-              className={'list-item'}
-            />
-          </Grid>
-          <Grid item xs={6}>
-            <ListItemText
-              primary={personClassJobTitle}
-              secondary={personType}
-              className={'list-item'}
-            />
-          </Grid>
+  return (
+    <ListItem
+      key={props.profile.AD_Username}
+      component={Link}
+      target="_blank"
+      to={`/profile/${props.profile.AD_Username}`}
+      className={'list-item'}
+    >
+      <ListItemAvatar>
+        {avatar ? (
+          <Avatar className={`avatar`} src={`data:image/jpg;base64,${avatar}`} alt="" />
+        ) : (
+          <Avatar>
+            <PersonIcon color="primary" />
+          </Avatar>
+        )}
+      </ListItemAvatar>
+      <Grid container alignItems="center" spacing={1}>
+        <Grid item xs={12} sm>
+          <ListItemText
+            primary={
+              props.profile.Title &&
+              props.profile.Title !== '' &&
+              props.profile.PersonType === 'fac'
+                ? // If the user has a title
+                  hasNickName
+                  ? // If the user has a title and a nickname
+                    props.profile.Title +
+                    ' ' +
+                    props.profile.fullName +
+                    ' (' +
+                    props.profile.NickName +
+                    ')'
+                  : // If the user has a title and no nickname
+                    props.profile.Title + ' ' + props.profile.fullName
+                : // If the user doesn't have a title
+                hasNickName
+                ? // If the user doesn't have a title but has a nickname
+                  props.profile.fullName + ' (' + props.profile.NickName + ')'
+                : // If the user doesn't have a title or a nickname
+                  props.profile.fullName
+            }
```

Comment:
I agree, I have gone back-and-forth with different ways of doing this, trying to explore how to make it the least confusing

Reply:
I think the first half of that can be removed as well because the list items will never have profile.personType === 'fac' 
They can only ever be students,